src/routes/view-content/[guideId]/[bibleSection]/library-resource-menu/SearchByResource.svelte

```diff
@@ -0,0 +1,145 @@
+                </div>
+                <div class="fixed top-16 flex h-[calc(100vh-79px)] w-full flex-col overflow-scroll bg-white">
+                    {#each $parentResources as parentResource}
+                        <button
+                            on:click={() => parentResourceSearch(parentResource)}
+                            class="mx-4 mb-4 flex h-12 items-center justify-center rounded-lg border border-[#EAECF0] p-2 text-sm"
+                            data-app-insights-event-name="library-resource-search-select-parent-resource-button-clicked"
+                            >{parentResource.displayName}</button
+                        >
+                    {/each}
+                </div>
+            </div>
+        {/if}
+    {:else if showResourceMenu}
+        <div class="relative z-[40] flex flex-col">
+            <div class="fixed top-0 h-28 w-full bg-white">
+                <div class="relative flex w-full justify-center pb-2 pt-4">
+                    <button
+                        on:click={backToParentResourceMenu}
+                        class="absolute left-4 flex w-1/4 justify-start"
+                        data-app-insights-event-name="library-resource-search-back-to-parent-resource-button-clicked"
+                        ><ChevronLeftIcon /></button
+                    >
+                    <h2>{currentParentResource?.displayName}</h2>
+                    <button
+                        on:click={closeResourcesMenu}
+                        class="absolute right-4 flex w-1/4 justify-end"
+                        data-app-insights-event-name="library-resource-search-close-menu-button-clicked"
+                        ><XMarkIcon /></button
+                    >
+                </div>
+                <div class="px-4">
+                    <SearchInput bind:searchQuery placeholder={'Search'} />
+                </div>
+            </div>
+            <div class="fixed top-28 flex h-[calc(100vh-193px)] w-full flex-col overflow-scroll bg-white">
+                {#if isLoading}
+                    <FullPageSpinner />
+                {:else}
+                    {#each resourceSearchResources as resourceSearchResource}
+                        <button
+                            on:click={() => resourceSelected(resourceSearchResource)}
+                            data-app-insights-event-name="library-resource-search-select-resource-button-clicked"
+                            class="mx-4 mb-4 flex h-12 items-center justify-center rounded-lg border border-[#EAECF0] p-2 text-sm"
+                            >{resourceSearchResource.displayName}</button
+                        >
+                    {/each}
+                    {#if resourceSearchResources.length >= 100}
+                        <button
+                            class="btn btn-primary mx-auto mb-4 w-auto px-4"
+                            on:click={() => {
+                                searchQuery = '';
```

Comment:
Is there a reason to clear the searchQuery here? Wouldn't we want the "Load More" to respect the current search query? For instance if I search for Genesis in one of the Study Notes types it's going to return > 100 results and wouldn't I want to hit "Load More" to pull up more of the Genesis results?